### ISSUE TITLE
No setter for property 'discardIdleConnectionsSeconds'

### ISSUE THREAD

**Issue by lcardito** [quartz-scheduler/quartz#264]
Hey guys,

when defining the property 

```org.quartz.dataSource.NAME.discardIdleConnectionsSeconds = 60```

as per documentation [ConfigDataSources](http://www.quartz-scheduler.org/documentation/quartz-2.x/configuration/ConfigDataSources.html), we're getting the following error:

```
Caused by: org.quartz.SchedulerException: Could not initialize DataSource: schedule
	at org.quartz.impl.StdSchedulerFactory.instantiate(StdSchedulerFactory.java:1042)
	at org.quartz.impl.StdSchedulerFactory.getScheduler(StdSchedulerFactory.java:1559)
	... 16 common frames omitted
```

Looks like the issue has been reported a while back on [JIRA](https://jira.terracotta.org/jira/browse/QTZ-499) with the workaround: 

```
org.quartz.dataSource.NAME.maxIdleTime = 60
```

Should the documentation be updated perhaps?

Thanks!

**Comment by zemian** [comment]
Can you share your quartz.properties that has this problem? I think only Hikari Pool supports this property.

**Comment by zemian** [comment]
This problem is very similar to #281. We can make additional changes to similar fix to handle this property as well.

**Comment by zemian** [comment]
Resolved ece2753f
